src/ipc/handler.c

```diff
@@ -1094,6 +1094,21 @@ static int ipc_glb_tplg_message(uint32_t header)
 	}
 }
 
+#ifdef CONFIG_DEBUG
+static int ipc_glb_test_message(uint32_t header)
+{
+	uint32_t cmd = iCS(header);
+
+	switch (cmd) {
+	case SOF_IPC_TEST_IPC_FLOOD:
```

Comment:
I think we can add some trace log with the IPC index id to help identify which IPC is missed. We can only tell how many IPC is missed, but which one is missed will help the debug

Reply:
@xiulipan that makes sense in the driver. the FW cant actually keep count because the IPC's may not coming in order.

Reply:
@ranj063 
I would like to make test IPC counts the index id as its payload. To help us debug the order or sequence issues.

Reply:
@xiulipan inline comments ? care to elaborate.

Reply:
@lgirdwood i think he meant he added some comments in the commit